### PR TITLE
Updated readme in SwiftUI client

### PR DESCRIPTION
**Pull request by dfreniche**
Added some more detail to SwiftUI Client README

### CODE REVIEW

**Review by ljhaywar — COMMENTED** [review]
Thanks so much for writing this up, Diego!!!  I added some suggestions - most are nits.

I didn't try the installation from scratch as I already have my environment set up.

File: inventory/clients/ios-swiftui/InventoryDemo/InventoryDemo.xcodeproj/xcshareddata/xcschemes/InventoryDemo.xcscheme [hunk 1]
```diff
@@ -70,13 +70,6 @@
             ReferencedContainer = "container:InventoryDemo.xcodeproj">
          </BuildableReference>
       </BuildableProductRunnable>
-      <EnvironmentVariables>
```
[the remark this answers is not in the document]
Comment (dfreniche) [reply inventory/clients/ios-swiftui/InventoryDemo/InventoryDemo.xcodeproj/xcshareddata/xcschemes/InventoryDemo.xcscheme:73]: Yep, let @am-MongoDB review it but my reasoning is that:
- putting secrets in the Scheme works well for local builds, not so much for CI builds
- hence the need to read that from a file
But if there's a way to do it via Scheme I'm all ears!

File: inventory/clients/ios-swiftui/README.md [hunk 2]
```diff
@@ -1,7 +1,50 @@
...
+
+![](img/inventory-demo-swiftui-schemes.png)
+
+This project has two Schemes:
+- CI: used for Continous Integration. In this case, a GitHUb action is used to build the app. You can check the [workflow here](https://github.com/mongodb-developer/realm-demos/actions/workflows/build.yml)
+- InventoryDemo: scheme used to run the app locally in Simulator, Device, etc.
+
+## Changing the Realm App Id
+
+This client uses Realm Sync to store data in the cloud. To do this, you need a Realm App Id so this client knows where to store data. [Instructions are here](https://github.com/mongodb-developer/realm-demos/tree/main/inventory#-create-an-atlas-cluster)
```
[the remark this answers is not in the document]
Comment (dfreniche) [reply inventory/clients/ios-swiftui/README.md:31]: Brilliant

File: inventory/clients/ios-swiftui/README.md [hunk 3]
```diff
@@ -1,7 +1,50 @@
...
+1. Select `InventoryDemo` [Scheme](#Schemes)
+1. Before running, [change the Realm Id in `realm-app-id.txt`](#changing-the-realm-app-id)
+1. Build and Run
+
+## Schemes
+
+![](img/inventory-demo-swiftui-schemes.png)
+
+This project has two Schemes:
+- CI: used for Continous Integration. In this case, a GitHUb action is used to build the app. You can check the [workflow here](https://github.com/mongodb-developer/realm-demos/actions/workflows/build.yml)
```
[the remark this answers is not in the document]
Comment (dfreniche) [reply inventory/clients/ios-swiftui/README.md:26]: 🤦‍♂️

File: inventory/clients/ios-swiftui/README.md [hunk 4]
```diff
@@ -1,7 +1,50 @@
...
+
+This client uses Realm Sync to store data in the cloud. To do this, you need a Realm App Id so this client knows where to store data. [Instructions are here](https://github.com/mongodb-developer/realm-demos/tree/main/inventory#-create-an-atlas-cluster)
+
+To set the realm app-id open the file `realm-app-id.txt` and put in there your Realm App Id.
+
+## Tests
+
+There are a few tests:
+- Unit Tests
+- Tests of SwiftUI
```
[the remark this answers is not in the document]
Comment (dfreniche) [reply inventory/clients/ios-swiftui/README.md:39]: I put them in a separate folder so you can run them in isolation from Xcode if wanted. They are Unit Tests that test SwiftUI Views.

File: inventory/clients/ios-swiftui/README.md [hunk 5]
```diff
@@ -1,7 +1,50 @@
...
+
+To run the tests from Xcode: ⌘ + U
+
+### Running tests from the Command Line
 
 If tests were working then something like this should work:
 
 ```
-xcodebuild -project InventoryDemo.xcodeproj -scheme "ci" -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 12 Pro Max,OS=14.5' -derivedDataPath './output' REALM_APP_ID='inventorysync-xlrrz' test
+xcodebuild -project InventoryDemo.xcodeproj -scheme "ci" -sdk iphonesimulator -destination 'platform=iOS Simulator,name=iPhone 12 Pro Max,OS=14.5' -derivedDataPath './output' test
```
[the remark this answers is not in the document]
Comment (dfreniche) [reply inventory/clients/ios-swiftui/README.md:49]: Seems like I was using iOS 14.5 but you are running with iOS 14.5. Will look if there's a way to set the `destination` as something that doesn't change

File: inventory/clients/ios-swiftui/README.md [hunk 6]
```diff
@@ -1,7 +1,50 @@
...
+- Tests of SwiftUI
+- UI Automation Tests
+
+To run the tests from Xcode: ⌘ + U
+
+### Running tests from the Command Line
 
 If tests were working then something like this should work:
 
 ```
```
[the remark this answers is not in the document]
Comment (dfreniche) [reply inventory/clients/ios-swiftui/README.md:48]: This key shortcut (⌘ + U) is to run tests from inside Xcode. Need to clarify maybe?

File: inventory/clients/ios-swiftui/README.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (dfreniche) [reply inventory/clients/ios-swiftui/README.md:49]: Fixed by https://github.com/mongodb-developer/realm-demos/pull/41

File: inventory/clients/ios-swiftui/README.md (hunk 6 above)
[the remark this answers is not in the document]
Comment (ljhaywar) [reply inventory/clients/ios-swiftui/README.md:48]: Ah, the line numbers got messed up. I was asking about what directory to run the xcodebuild command from

[the remark this answers is not in the document]
Comment (dfreniche) [reply inventory/clients/ios-swiftui/README.md:48]: Fixed, hopefully!

**Pull request merged**